test/content/audio/approaching_test.exs

```diff
@@ -9,30 +9,33 @@ defmodule Content.Audio.ApproachingTest do
 
       assert Content.Audio.to_params(audio) ==
                {:canned,
-                {"112", spaced(["896", "892", "920", "910", "901", "21014"]), :audio_visual}}
+                {"114", spaced(["896", "892", "920", "910", "901", "21014", "925"]),
+                 :audio_visual}}
```

Comment:
Did we not already have a test case in this file for the "four cars" message? I don't see any reference to that particular take, 922.

Reply:
Not in this file. Generally we've been preferring the higher level behavior tests because they give a lot more coverage and resilience for about the same amount of code. `realtime_test.exs` has the cases for that message.